### ISSUE TITLE
LethalLevelLoader compatibility

### ISSUE THREAD

**Issue by Qugar** [Sylkadi/BrutalCompanyMinus#17]
Since version 0.8.3, the modded moons of LethalLevelLoader no longer work with BrutalCompanyMinus. You can no longer land on the moons.

**Comment by Sylkadi** [comment]
I can't replicate at all, If you give me the code i could figure out if it's my mod or not that's causing the issue
![Screenshot_148](https://github.com/Sylkadi/BrutalCompanyMinus/assets/101504347/2ff87f2c-3a27-4a9d-8555-47f29d5a2786)

**Comment by Qugar** [comment]
Thanks for checking out.

If you update your mod here, you should no longer be able to land on modded moons, but with the old version that is currently still installed, it works without any problems. But of course it could be that something else gets in the way.

018db4cd-8945-5523-6e1f-df7a792af5fc

**Comment by Sylkadi** [comment]
I updated the mod and I was able to land on every modded moon, huh?

**Comment by Qugar** [comment]
I'm confused. It works now, but I haven't changed anything.

Hmm, I'll set the config and then test it again.
I'll give you feedback again and close the thread if it works.

Sry, that's kind of weird but thanks for looking.

**Comment by Qugar** [comment]
Yes, everything works now, I have no idea what went wrong yesterday.

Sorry for wasting your time and thanks again for checking.

**Comment by Sylkadi** [comment]
If it happens again, send me the log file, cause it might of been an event that caused it, otherwise I don't know

**Comment by Qugar** [comment]
Now it happened again, I think the error starts in the log in line 2837.

[LogOutput.log](https://github.com/Sylkadi/BrutalCompanyMinus/files/14320145/LogOutput.log)


Edit: Is this a problem with LethalLevelLoader? Am I reading correctly that it apparently didn't read the moon when starting?

If so, I must have been very unlucky yesterday because it didn't work several times after updating your mod.
Or could that be related to your mod?

Sorry for the stupid questions, just if I recognized it correctly then I might recognize it next time and be able to tell the difference.

**Comment by Sylkadi** [comment]
It should not longer break in latest release, but something is still going wrong, so i wont close.

**Comment by Qugar** [comment]
I haven't had any problems with the latest version so far.

If you want, I can give you feedback again after I have had a longer gaming session.

Edit: So during today's four hour test there were absolutely no problems.

**Comment by apeng0907** [comment]
[Info   :   BepInEx] Loading [BrutalCompanyMinus 0.9.3]
[Error  : Unity Log] The AssetBundle 'IO.Stream' can't be loaded because another AssetBundle with the same files is already loaded.
[Info   :BrutalCompanyMinus] BrutalCompanyMinus 0.9.3 is done patching.
